docs/csharp/language-reference/statements/snippets/checked-and-unchecked/Program.cs

```diff
@@ -9,14 +9,14 @@ void MainExample()
 
     unchecked
     {
-        Console.WriteLine(a + 1);  // output: 0
+        Console.WriteLine(a + 3);  // output: 2
```

Comment:
Want to show that overflow doesn't falls back to the minimum possible value, but really wraps